runtime/src/bank.rs

```diff
@@ -6923,6 +6924,13 @@ impl Bank {
         Ok(sanitized_tx)
     }
 
+    pub fn fully_verify_transaction(
+        &self,
+        tx: VersionedTransaction,
+    ) -> Result<SanitizedTransaction> {
+        self.verify_transaction(tx, TransactionVerificationMode::FullVerification)
```

Comment:
i love type safety. but it's too verbose to impose this fn signature at call-site, imo.

Reply:
also, these small wrapper fns avoid new `use TransactionVerificationMode`-ing only for the sake of this fn invocation at call-site too